<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="otv">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(255, 203, 81, 237), stop:0.886364 rgba(255, 134, 73, 255));
border: 2px solid #ff7474;
border-radius: 5px;
font: 75 28pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="task">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>1111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(152, 29, 29);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1021</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 25pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Ответье на ряд вопросов для определения вашего уровня</string>
   </property>
  </widget>
  <widget class="QLabel" name="anc_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="anc_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="anc_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>210</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="anc_4">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>210</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="a">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(173, 0, 0);</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="b">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(173, 0, 0);</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QLabel" name="c">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(173, 0, 0);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="d">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>210</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(173, 0, 0);</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Выберите ваш вариант:</string>
   </property>
  </widget>
  <widget class="QLabel" name="chet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(173, 0, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="chet_2">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>130</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(173, 0, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;</string>
   </property>
   <property name="text">
    <string>Текущее количество баллов:</string>
   </property>
  </widget>
  <widget class="QLabel" name="chet_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(173, 0, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="prog">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: center;
border-radius: 6px;
background-color: rgb(172, 255, 172);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="con">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>711</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(255, 203, 81, 237), stop:0.886364 rgba(255, 134, 73, 255));
border: 2px solid #ff7474;
border-radius: 10px;
font: 75 16pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Готово!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Время:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1621</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(149, 255, 247, 237), stop:0.886364 rgba(201, 107, 255, 255));</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cont">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>711</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.227273 rgba(255, 203, 81, 237), stop:0.886364 rgba(255, 134, 73, 255));
border: 2px solid #ff7474;
border-radius: 10px;
font: 75 18pt &quot;Comic Sans MS&quot;;</string>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>1061</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#d8fffb;
border: 2px solid #ff7474;
border-radius: 10px;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>lineEdit</zorder>
  <zorder>otv</zorder>
  <zorder>task</zorder>
  <zorder>label_2</zorder>
  <zorder>anc_1</zorder>
  <zorder>anc_2</zorder>
  <zorder>anc_3</zorder>
  <zorder>anc_4</zorder>
  <zorder>a</zorder>
  <zorder>b</zorder>
  <zorder>c</zorder>
  <zorder>d</zorder>
  <zorder>label_11</zorder>
  <zorder>chet</zorder>
  <zorder>chet_2</zorder>
  <zorder>label_12</zorder>
  <zorder>chet_3</zorder>
  <zorder>prog</zorder>
  <zorder>con</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>cont</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
